clients/fides-js/src/components/fides.css

```diff
@@ -317,23 +317,6 @@ div#fides-modal .fides-modal-button-group {
   background: var(--fides-overlay-hover-color);
 }
 
-.fides-back-button {
```

Comment:
I hit the 20kb limit for `fides-js` 😵 so I went and looked through the CSS file for classes that aren't used anymore. that reduced by a whole .2kb!

Reply:
our CSS classes are all in this file. it might make sense to have a separate TCF css file which only `TcfOverlay` imports. that should reduce our vanilla `fides-js` even further (FYI @galvana , who has been doing a ton of CSS work!)